vulcan/lib/server/controllers/workflows_controller.rb

```diff
@@ -15,12 +15,11 @@ def fetch
   def submit
     # This is a stub API that will be used for development only.
     # Submit input data and get the specified output status back.
-
     raise Etna::NotFound, "No data for workflow #{@params[:workflow_name]}." unless "umap" == @params[:workflow_name]
 
     success(File.read(File.join(
       File.dirname(__FILE__),
-      "../data/#{@params[:status]}.json")), 'application/json')
+      "../../../data/#{@params[:status]}.json")), 'application/json')
```

Comment:
For a real API, I think this should actually just become `/api/:project_name/workflows/:workflow_name`...I was trying to get different status JSONs back, but not sure how to mock that out...

Reply:
Yessir.  I intend to change this section here soon anyways.